### ISSUE TITLE
Compile failed on macOS if no fontconfig installed

### ISSUE THREAD

**Issue by 7sDream** [servo/libfontconfig#53]
I notice that `build.rs` will try to find installed `fontconfig` library and dynamic link to it if found. 

When not found, it will use make to build static library itself.

But the makefile.cargo do nothing in macOS, this leads to build failed.

Is this expected behavior?

Related: 7sDream/fontfor#10

**Comment by jdm** [comment]
It's expected in that Servo only uses fontconfig on linux, so we never bothered to make the build work for macOS.

**Comment by 7sDream** [comment]
Oh, it's sound reasonable.

So what do Servo use to find/match fonts in macOS? Apple's Core-Text API?

**Comment by jdm** [comment]
Exactly. Specifically, using a hugher-level crate like font-kit which has platform-specific backends and uses core-text for macOS.
